frontend/cypress/integration/queryBuilder.spec.js

```diff
@@ -168,7 +168,18 @@ describe('The Query Builder', () => {
     // TODO: create test
   });
 
-  xit('makes a copy of a dataset', () => {
-    // TODO: create test
+  it('makes a copy of a dataset', () => {
+    cy.fixture('datasets').then((datasets) => {
+      cy.intercept('api/datasets/mine/', datasets);
```

Comment:
I need you to test that this button works for published datasets as well.

Reply:
Had not  seen this